spec/amber/router/params_spec.cr

```diff
@@ -0,0 +1,84 @@
+require "../../../spec_helper"
+
+module Amber::Router
+  describe Parser do
+    headers = HTTP::Headers.new
+
+    context "when parsing query string" do
+      it "returns query params" do
+        request = HTTP::Request.new("GET", "/?test=test")
+        request.params["test"].should eq "test"
+      end
+
```

Comment:
@eliasjpr it seems that here a test is missing to show what happens when request.params["test"] is called but contains multiple values.

Reply:
See next test below. Let me know if that answers your question👇

Reply:
In the next one you call fetch_all(), not params[]. I am saying that it seems a test is missing where you do fill it with multiple params, but then call params[] instead of fetch_all.

Reply:
I don't get this. `request.params["test"].should eq "test2"` in that case. Is that a thing that needs retested?

Reply:
I see what you're saying, I have updated the test below